changes/5341.deprecated

```diff
@@ -0,0 +1 @@
+Deprecated the `plugins` key under the `/api/status/` REST API endpoint. Refer to `nautobot-apps` instead.
```

Comment:
This is a backwards incompatible change? Should we provide the old behavior in API version 2.1?

Reply:
Sorry, I see now that it is not a breaking change.